.github/workflows/trigger_athena.yml

```diff
@@ -0,0 +1,15 @@
+name: Trigger Athena build
```

Comment:
I'd suggest not to use `trigger` in the name of the test and file. That may suggest this is related to the Athena trigger.

Reply:
I see. It's literally called a "pipeline trigger" in GitLab 😅

Reply:
Can we keep this? I think it's standard terminology.